face/src/helpers/or.js

```diff
@@ -1,3 +1,4 @@
-export default function(a, b) {
-    return a || b;
+export default function(...args) {
+    args.pop();
+    return args.reduce((a, b) => !!(a || b), false);
```

Comment:
no need to make this so complex.
When you have more than 2 operands, you should move this logic somewhere else.